src/pages/layout/Search/SearchAccount.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import {useGetAccountResources} from "../../../api/hooks/useGetAccountResources";
+import ResultLink from "./ResultLink";
+
+type SearchAccountProps = {
+  address: string;
+};
+
+export default function SearchAccount({address}: SearchAccountProps) {
+  const {isLoading, error, data} = useGetAccountResources(address);
+
+  if (isLoading || error || !data) {
```

Comment:
we can show a spinner when isLoading https://github.com/aptos-labs/explorer/blob/main/src/pages/Governance/Stake/components/CreateOrEdit.tsx#L36

Reply:
I'm not sure if it's the best idea here even though showing spinner is a common practice in similar scenarios. The reason being: `isLoading` will be true for a while (5-10 seconds) when the txnHash or address is invalid.